pkg/apis/extensions/types.go

```diff
@@ -638,3 +638,41 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// PodSelector is a label query over a set of API objects. MatchLabels and
+// MatchExpressions are ANDed. An empty PodSelector matches all objects. A
+// null PodSelector matches no objects.
+type PodSelector struct {
+	// MatchLabels is a map of key value pairs. A single key value in the
+	// map is equivalent to a PodSelectorRequirement where the Key is the key,
```

Comment:
The Go type names aren't part of the API and users shouldn't be expected to know them. Use the json field name.